recurring_ical_events.py

```diff
@@ -319,7 +364,7 @@ def all(self):
         I personally do not recommend to use this because
         this method is not documented and you may end up with lots of events most of which you may not use anyway."""
         # MAX and MIN values may change in the future
-        return self.between((1970, 1, 1), (2038, 1, 1))
```

Comment:
Why is it even there then?

Reply:
The tests use all() so we can test that all events are retrieved easily.

Reply:
Well... then it is a function of the test, not the API.